src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/impl/ActionSvcRestClientImpl.java

```diff
@@ -126,6 +129,41 @@ public ActionPlanDTO createActionPlan(
     return responseEntity.getBody();
   }
 
+  @Retryable(
+      value = {RestClientException.class},
+      maxAttemptsExpression = "#{${retries.maxAttempts}}",
+      backoff = @Backoff(delayExpression = "#{${retries.backoff}}"))
+  @Override
+  public List<ActionPlanDTO> getActionPlansBySelectorsSocial(final String collectionExerciseId) {
+    log.debug(
+        "Retrieving action plan for selectors, collectionExerciseId: {}", collectionExerciseId);
+
+    MultiValueMap<String, String> queryParams = new LinkedMultiValueMap<>();
+    queryParams.add(COLLECTION_EXERCISE_ID, collectionExerciseId);
+    UriComponents uriComponents =
+        restUtility.createUriComponents(appConfig.getActionSvc().getActionPlansPath(), queryParams);
+
+    ResponseEntity<List<ActionPlanDTO>> responseEntity;
+    try {
+      responseEntity =
+          restTemplate.exchange(
+              uriComponents.toString(),
+              HttpMethod.GET,
+              null,
+              new ParameterizedTypeReference<List<ActionPlanDTO>>() {});
+    } catch (HttpClientErrorException e) {
+      if (e.getStatusCode() == HttpStatus.NOT_FOUND) {
+        return null;
+      }
+      throw e;
+    }
+
+    log.debug(
+        "Successfully retrieved action plan for selectors, collectionExerciseId: {}",
```

Comment:
Would it be worth logging the `ActionPlanID` here? To just confirm the one received?